### ISSUE TITLE
I have a new VR display. Will to give equity in my company for someone that can use this

### ISSUE THREAD

**Issue by Tom12121212** [sencercoltu/steamvr-undistort#13]
I have designed a new VR display. 220 Degree FOV. You cant see the edges of the screens. Dual 4k at over 90 FPS. This is really a message for sencercoltu. But if anyone that can use this code I am willing to give equity in my company and pay them. I was the co designer of the first Oculus Rift prototype. You can search for me Thomas Seidl. Yes there was more than just Palmer Luckey. I told him the lens design and other things. 

I use fresnel lenses. You can write to me at 
       tseidl    at   sharevr  
with the usual com at the end.

**Comment by Tom12121212** [comment]
PS synthead You seem to be the guy now. Please see my above post. I used to have a memory Moog, Jupiter 6, 808, Custom Analog machine etc

**Comment by sencercoltu** [comment]
This tool won't work unless your headset uses SteamVR Tracking HDK (or a compatible tech), and you need to be able to read and write the coefficients from the headset. (e.g. Pimax, which uses the HDK but ignores the coefficients, and Oculus, won't work with this tool.)

**Comment by Tom12121212** [comment]
Yes we use SteamVR tracking with steamVR controllers. We only need it working on Steam to start with.  Our design is similar to Pimax with a few secret extra features. But the optics perform similar. Main requirement is that it will perform without much lag.

When you say ignores the coefficients you mean ignores the standard picture stretch? I presume Pimax has some picture distortion correction? You got time for some txt chat on skype back and forth?

**Comment by sencercoltu** [comment]
> Yes we use SteamVR tracking with steamVR controllers. We only need it working on Steam to start with. Our design is similar to Pimax with a few secret extra features. But the optics perform similar. Main requirement is that it will perform without much lag.
> 
> When you say ignores the coefficients you mean ignores the standard picture stretch? I presume Pimax has some picture distortion correction? You got time for some txt chat on skype back and forth?

You can contact me on skype, same username.
